Google.Api.Generator.Tests/ProtoTests/ChildResource/Testing.ChildResource/ChildResourceResourceNames.g.cs

```diff
@@ -22,67 +22,178 @@ namespace Testing.ChildResource
     /// <summary>Resource name for the <c>Project</c> resource.</summary>
     public sealed partial class ProjectName : gax::IResourceName, sys::IEquatable<ProjectName>
     {
-        private static readonly gax::PathTemplate s_template = new gax::PathTemplate("projects/{project}");
+        /// <summary>The possible contents of <see cref="ProjectName"/>.</summary>
+        public enum ResourceType
+        {
+            /// <summary>A resource of an unknown type.</summary>
+            Unknown = 0,
+
+            /// <summary>A resource of type 'project'.</summary>
```

Comment:
I think we'll need to look carefully at the descriptions here, and possibly the enum name as well. The resource is the same type in all cases, it's just how it's named that differs.

Reply:
Changed as discussed.